srcpkgs/mle/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'mle'
 pkgname=mle
-version=1.4.3
+version=1.5.0
 revision=1
 build_style=gnu-makefile
 hostmakedepends="libtool"
-makedepends="termbox-devel lua53-devel pcre-devel uthash"
+makedepends="termbox-devel lua54-devel pcre-devel uthash"
```

Comment:
```suggestion
makedepends="termbox-devel lua54-devel pcre2-devel uthash"
```

Reply:
The build fails with this change. https://github.com/void-linux/void-packages/actions/runs/3245745329/jobs/5323669623

Moreover, pcre2-devel does not exist in void repos.

Reply:
I see, I looked at master which supports pcre2, but it's not in this release yet, my bad. pcre2-devel does exist in void repositories though.